cmd/geth/flags.go

```diff
@@ -152,8 +152,37 @@ var (
 	}
 	LogDirFlag = DirectoryFlag{
 		Name:  "log-dir,logdir",
-		Usage: "Directory in which to write log files.",
-		Value: DirectoryString{filepath.Join(common.DefaultDataDir(), "logs")},
+		Usage: "Directory in which to write log files",
+		Value: DirectoryString{filepath.Join(common.DefaultDataDir(), "<chain>", "logs")},
+	}
+	LogMaxSizeFlag = cli.IntFlag{
+		Name:  "log-max-size,log-maxsize",
+		Usage: "Maximum size of a single log file (in bytes)",
+		Value: 0,
+	}
+	LogMinSizeFlag = cli.IntFlag{
+		Name:  "log-min-size,log-minsize",
+		Usage: "Minimum size of a log file, to be considered for log-rotation (in bytes)",
+		Value: 0,
+	}
+	LogMaxTotalSizeFlag = cli.IntFlag{
+		Name:  "log-max-total-size,log-maxtotalsize",
+		Usage: "Maximum total size of all (current and archived) log files (in bytes)",
+		Value: 0,
+	}
+	LogIntervalFlag = cli.StringFlag{
+		Name:  "log-interval",
+		Usage: "Log rotation interval, one of values: never, hourly, daily, weekly, monthly",
+		Value: "never",
+	}
+	LogMaxAgeFlag = cli.StringFlag{
+		Name:  "log-max-age,log-maxage",
+		Usage: "Maximum age of the oldest log file, valid time units: 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'",
```

Comment:
Can we also add `'d'` and `'w'` for days and weeks? For me at least that would be convenient ;)

Reply:
And wait a second... max age of oldest log file in nanoseconds?

Reply:
The units are copied from documentation of `time.ParseDuration` function. I'll change this, i think 'h', 'd', 'w' should be sufficient. What do you think?

Reply:
Yea, I think those'll be fine